### DISCUSSION TITLE
Cluster and public network, two network card

### DISCUSSION THREAD

**Discussion by ldapomni** [seaweedfs/seaweedfs#2517] [Q&A]
How to run seaweedfs if two network cards are used in servers to organize a cluster network for exchange between vol and client networks. An analogue of the public network and cluster network in ceph? That is, so that replication and rebalance go through one network card, and work with the client through another? (-ip, -ip.bind, -publicUrl,...?)

**Comment by chrislusf** [comment]
The seaweedfs servers themselves do not know the difference. Each server has its own ip address or host name.

The clients can send traffic to the servers by `-publicUrl`, which is just an uninterpreted string to the servers.
